.fog.example

```diff
@@ -0,0 +1,12 @@
+#####################################################
+# This is configuration snippet to configure Google Cloud Platform
+# for fig library.
+#
+# 1. Copy this file into your home dir: "~/.fog"
+# 2. Follow instructions to generate a private key: 
+#   https://cloud.google.com/storage/docs/authentication#generating-a-private-key
+
+my_credential:
```

Comment:
Although I do admit that's our doc issue, I'm not sure if `my_credential` will actually work by default if you didn't explicitly load them via Fog::Credentials. AFAIK only "default" works by default and whatever is in "test" is loaded by tests. 

Have you tested this?